docs/utils/script.py

```diff
@@ -178,10 +177,10 @@ def append_doc_to_spec_file(file_index: dict):
     # Appending md doc into skeleton spec
     spec['tags'] = []
     spec['x-tagGroups'] = []
-    for tag_name, md_files in file_index.items():
+    for module, md_files in index.items():
         tag_names = []
         for md_file in md_files:
-            name = get_origin_file_name(md_file)
+            name = truncate_md_extension(remove_module_prefix_from_file(md_file, module))
```

Comment:
New setup, only sanitize file and don't rename file, hence overview file will work as expected